README.md

```diff
@@ -11,18 +11,18 @@ var server = new TChannel({host: '127.0.0.1', port: 4040});
 var client = new TChannel({host: '127.0.0.1', port: 4041});
 
 // normal response
-server.register('func 1', function (arg1, arg2, peerInfo, cb) {
-    console.log('func 1 responding immediately 1:' + arg1.toString() + ' 2:' + arg2.toString());
+server.register('func 1', function (reqHead, reqBody, peerInfo, cb) {
```

Comment:
the protocol is actually designed for [label, head, body]. Its fine to expose that more clearly in all the documentation.